### DISCUSSION TITLE
Can a Playbook call other Playbooks?

### DISCUSSION THREAD

**Discussion by btzq** [semaphoreui/semaphore#1940] [Q&A] [answered]
We have a service that has 5 microservices. 

Hence, we have 5 Playbooks. Each Playbook is a task.

We want to create a 6th Task, that runs playbook 1-5 in order. 

Sort of like a Master Task/Playbook. 

Is this possible?

**Comment by bodzio4749** [comment, marked as answer]
I hope I am missing something, but Isn't this a standard feature of the ansible-playbook program itself?  Import_playbook, include_playbook, import_task, include_task and finally roles. All of the modules are used to call each other. Replicating ansible features into semaphore itself may create more problems than solutions. 
Again, if I misunderstood something I apologize.
